client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-maps-panel/incident-maps-panel.component.ts

```diff
@@ -108,25 +108,13 @@ export class IncidentMapsPanel implements OnInit {
 
   async generateMapRequest(mapLink, fileName) {
     const url = mapLink;
-    let response;
 
     try { 
-      await this.capacitorService.isMobile.then(isMobile => {
-        if (isMobile) {
-          const options = {
-            method: 'GET',
-            url,
-            params: {responseType: 'blob'}   
-          };
-          response = CapacitorHttp.get(options).then(resp => {
-              this.downloadFile(resp.data, fileName);
-              this.snackbarService.open('PDF downloaded successfully.', 'Close', {
-                duration: 10000,
-                panelClass: 'snackbar-success-v2',
-              });
-          });
+      await this.capacitorService.isMobile.then(async isMobile => {
```

Comment:
MInor, but some considerations regarding readbility, may be better to use:

const isMobile = await this.capacitorService.isMobile();
if (isMobile) { 
....
}
else {
....
{

Reply:
this.capacitorService.isMobile returns a Promise so I think **then** has to be used here. If you look at where it's called elsewhere in the app, that's how it gets called. I removed the async keyword as it's not needed